modules/admin-ui/src/main/webapp/scripts/shared/partials/hotkeyCheatSheet.html

```diff
@@ -0,0 +1,28 @@
+<div ng-class="{'modal-animation modal-overlay': helpVisible}"></div>
+<div ng-controller="HotkeyCheatSheetCtrl" ng-show="helpVisible" class="modal modal-animation">
+  <header>
+    <a class="fa fa-times close-modal" ng-click="toggleCheatSheet()"></a>
+    <h2 translate="HOTKEYS.CHEAT_SHEET.TITLE"><!-- Keyboard Shortcuts --></h2>
+  </header>
+  <div class="modal-content active"><div class="modal-body"><div class="full-col">
+    <div class="obj tbl-list" ng-repeat="group in keyGroups">
+      <header translate="{{ 'HOTKEYS.GROUPS.' + group.name | uppercase }}" class="no-expand">
+      </header>
+      <table class="main-tbl"><tbody>
+        <tr ng-repeat="hotkey in group.keys">
+          <td class="hotkey">
+            <p class="combo" ng-repeat="combo in hotkey.combos">
+              <span ng-repeat="chord in combo" class="cfp-hotkeys-keys">
+                <span ng-repeat="key in chord">
+                  <span class="cfp-hotkeys-key" translate="{{ 'HOTKEYS.KEYS.' + key | uppercase }}" translate-default="{{ key | uppercase }}"></span>
+                  {{ $last ? '' : '+' }}
+                </span>
+              </span>
+            </p>
+          </td>
+          <td translate="{{ 'HOTKEYS.DESCRIPTIONS.' + hotkey.description | uppercase }}"></td>
+        </tr>
+      </tbody></table>
+    </div>
+  </div></div>
```

Comment:
That looks weird (both div on same line)

Reply:
At least it is consistent with the above. :wink: 

By the way, what about the `<table><tbody>` in one line, is that okay? ^_^ \*is shoveling his own grave, basically\*